src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -31,7 +31,7 @@ public static TestCommand FromArgs(string[] args, string msbuildPath = null)
             var msbuildArgs = new List<string>()
             {
                 "-target:VSTest",
-                "-verbosity:quiet",
+                "-verbosity:minimal",
```

Comment:
Check what is impact of [PR](https://github.com/dotnet/cli/pull/9267/files#diff-ab184f453b5f0090121f80776937e88bR63) on this change.